File: BoletosAPI/Models/Secciones.cs
namespace BoletosAPI.Models
{
    public class Secciones
    {
        public int SeccionId { get; set; }
        public int EventoId { get; set; }
        public string Seccion { get; set; } = String.Empty;
        public double Precio { get; set; }
    }
}

File: BoletosDesktop/Models/Secciones.vb
Public Class Secciones
    Property SeccionId As Integer
    Property EventoId As Integer
    Property Seccion As String
    Property Precio As Double

    Public Sub New()

    End Sub
    Public Sub New(seccion As String, precio As Double)
        Me.Seccion = seccion
        Me.Precio = precio
    End Sub
End Class
